deployments/loader/templates/deployment.yaml

```diff
@@ -1,14 +1,21 @@
+{{- if .Values.runOnce }}
```

Comment:
Do we need the change to `values.yaml`?

Reply:
`runOnce` is a loader parameter, so it's already defined in the `values.yaml` and read from the ConfigMap template.